js/src/pages/edit-paid-ads-campaign/edit-paid-ads-campaign-form.js

```diff
@@ -66,7 +66,7 @@ const EditPaidAdsCampaignForm = ( props ) => {
 							) }
 							description={ createInterpolateElement(
 								__(
-									'Paid Smart Shopping campaigns are automatically optimized for you by Google. <link>See what your ads will look like.</link>',
+									'Paid ad campaigns are automatically optimized for you by Google. <link>See what your ads will look like.</link>',
```

Comment:
I have added a new line [0a66282](https://github.com/woocommerce/google-listings-and-ads/pull/1342/commits/0a662826903ad2765ccbb11d07b011db721d4589)

Reply:
Thanks for the change, visually it looks good to me:
![image](https://user-images.githubusercontent.com/11388669/159521306-338f6d83-664b-4ca6-b8b2-0a3648a46d1c.png)

Although I'm can't seem to find any similar examples of where we are inserting a new line like that. @eason9487 @ecgan would either of you be able to confirm if 0a66282 is the right way to add a newline?

Reply:
I suggest doing something like this: 

```jsx
description={
	<>
		{ __(
			'Paid ad campaigns are automatically optimized for you by Google.',
			'google-listings-and-ads'
		) }
		<br />
		<AppDocumentationLink
			context="edit-ads"
			linkId="see-what-ads-look-like"
			href="https://support.google.com/google-ads/answer/6275294"
		>
			{ __(
				'See what your ads will look like.',
				'google-listings-and-ads'
			) }
		</AppDocumentationLink>
	</>
}
```

Reasons: 

- The intention is to have two physical lines, and with this, we do have two separated i18n strings, instead of one long i18n string containing `<br>` in it.
- With this approach, we don't need `createInterpolateElement` (remember to remove the import at the top of the file).